### DISCUSSION TITLE
How to access notebook name in JupyterLab without ipylab?

### DISCUSSION THREAD

**Discussion by liquidcarbon** [jtpio/ipylab#139] [Q&A] [answered]
ipylab is wonderful, just wanted to minimize dependencies :) 

context: https://github.com/jupyter/notebook/issues/7293

**Comment by jtpio** [comment, marked as answer]
@liquidcarbon you can try reading the `JPY_SESSION_NAME` environment variable with:

```py
import os
os.environ.get('JPY_SESSION_NAME')
```

![image](https://github.com/jtpio/ipylab/assets/591645/4f239577-e666-456e-93a8-68ed9a4979f9)

**Reply by liquidcarbon** [reply]
wonderful, thank you!
